File: 236A/boyOrGirl.cpp
#include "string"
#include "iostream"
#include "unordered_set"
using namespace std;
int main()
{
    string s;
    cin >> s;
    unordered_set<char> name;
    int c = 0;
    for(auto ch : s)
    {
        if (name.find(ch) == name.end())
        {
            c++;
        }
        name.insert(ch);
    }

    if (c%2 == 0)
    {
        cout<< "CHAT WITH HER!";
    }
    else
    {
        cout<< "IGNORE HIM!";
    }
    
    
    
    return 0;
}

File: 236A/boyOrGirl.java
import java.util.HashSet;
import java.util.Scanner;
import java.util.Set;
/**
 * boyOrGirl
 */
public class boyOrGirl {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        Set<Character> ch = new HashSet<>();
        String username = in.nextLine();
        for (int i = 0; i < username.length(); i++) {
            ch.add(username.charAt(i));
        }
        if (ch.size() % 2 == 0) {
            System.out.println("CHAT WITH HER!");
        } else {
            System.out.println("IGNORE HIM!");
        }
        in.close();
    }
}